text/0000-Project-Goals-2025h1.md

```diff
@@ -0,0 +1,662 @@
+[wg-diagnostics]: https://rust-lang.github.io/compiler-team/working-groups/diagnostics/
+[wg-embedded]: https://github.com/rust-embedded/wg
+[wg-embedded-core]: https://www.rust-lang.org/governance/teams
+[wg-embedded-cortex-a]: https://www.rust-lang.org/governance/teams
+[wg-embedded-cortex-m]: https://www.rust-lang.org/governance/teams
+[wg-embedded-cortex-r]: https://www.rust-lang.org/governance/teams
+[wg-embedded-hal]: https://www.rust-lang.org/governance/teams
+[wg-embedded-infra]: https://www.rust-lang.org/governance/teams
+[wg-embedded-libs]: https://www.rust-lang.org/governance/teams
+[wg-embedded-linux]: https://www.rust-lang.org/governance/teams
+[wg-embedded-msp430]: https://www.rust-lang.org/governance/teams
+[wg-embedded-resources]: https://www.rust-lang.org/governance/teams
+[wg-embedded-riscv]: https://www.rust-lang.org/governance/teams
+[wg-embedded-tools]: https://www.rust-lang.org/governance/teams
+[wg-embedded-triage]: https://www.rust-lang.org/governance/teams
+[wg-ffi-unwind]: https://github.com/rust-lang/project-ffi-unwind
+[wg-gamedev]: https://github.com/rust-gamedev
+[wg-gcc-backend]: https://github.com/rust-lang/rustc_codegen_gcc
+[wg-incr-comp]: https://www.rust-lang.org/governance/teams
+[wg-inline-asm]: https://github.com/rust-lang/project-inline-asm
+[wg-leads]: https://www.rust-lang.org/governance/teams
+[wg-llvm]: https://rust-lang.github.io/compiler-team/working-groups/llvm/
+[wg-macros]: https://github.com/rust-lang/wg-macros
+[wg-mir-opt]: https://rust-lang.github.io/compiler-team/working-groups/mir-opt/
+[wg-parallel-rustc]: https://rust-lang.github.io/compiler-team/working-groups/parallel-rustc/
+[wg-polonius]: https://rust-lang.github.io/compiler-team/working-groups/polonius/
+[wg-polymorphization]: https://rust-lang.github.io/compiler-team/working-groups/polymorphization/
+[wg-prioritization]: https://rust-lang.github.io/compiler-team/working-groups/prioritization/
+[wg-rust-by-example]: https://github.com/rust-lang/rust-by-example
+[wg-rustc-dev-guide]: https://rust-lang.github.io/compiler-team/working-groups/rustc-dev-guide/
+[wg-rustc-reading-club]: https://rust-lang.github.io/rustc-reading-club/
+[wg-safe-transmute]: https://github.com/rust-lang/project-safe-transmute
+[wg-secure-code]: https://github.com/rust-secure-code/wg
+[wg-security-response]: https://github.com/rust-lang/wg-security-response
+[wg-self-profile]: https://rust-lang.github.io/compiler-team/working-groups/self-profile/
+[wg-triage]: https://www.rust-lang.org/governance/teams
+[windows]: https://www.rust-lang.org/governance/teams
+
+
+[Bastian Kersting]: https://github.com/1c3t3a
+[Boxy]: https://github.com/BoxyUwU
+[Scott Schafer]: https://github.com/Muscraft
+[Sparrow Li]: https://github.com/SparrowLii
+[Manuel Drehwald]: https://github.com/ZuseZ4
+[Alejandra González]: https://github.com/blyxyas
+[Celina G. Val]: https://github.com/celinval
+[Connor Horman]: https://github.com/chorman0773
```

Comment:
May not be needed any longer given we are removing the specification testing goal.